homeassistant/package_constraints.txt

```
@@ -145,3 +145,7 @@ matplotlib==3.6.1
 # pyOpenSSL 23.0.0 or later required to avoid import errors when
 # cryptography 39.0.0 is installed with botocore
 pyOpenSSL>=23.0.0
+
+# The get-mac package has been replaced with getmac. Installing get-mac alongside getmac
+# breaks getmac due to them both sharing the same python package name inside 'getmac'.
+get-mac<0.0.0
```

Please run the `gen_requirements_all.py` script to update the requirements.

---

I have run the gen requirements script, but because the get-mac constraint is a negative one, ie never allow get-mac to be included, and now that no core integrations use it anymore, the gen requirements script is not updating the requirements txt files with get-mac, only the vilfo-api-client one.

---

Try setting it to a high number instead like we do above.

```suggestion
get-mac==1000000000.0.0
```

---

Good idea, done.